cmd/neofs-cli/internal/client/sdk.go

```diff
@@ -15,8 +16,23 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	// max GRPC message size.
+	defaultBufferSize = 4 * 1024 * 1024 // 4MB
+)
+
 var errInvalidEndpoint = errors.New("provided RPC endpoint is incorrect")
 
+var buffers *sync.Pool
```

Comment:
Do we need it for _cli_ at all? It's a short-lived thing that usually talks to a single server, it won't benefit from this in any way.

Reply:
Good question, my opinion is no. But if you have the same question, this means also no. Removed